### PR TITLE
Added --use-virtualenvs option to allow choosing envs to run

### PR DESCRIPTION
**Pull request by AlexejPenner**
## Describe changes
With this you should be able to run integration tests locally without separate virtualenvs for each test. This is necessary in case you installed your virtualenvironment with venv or pyenv. 

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -249,61 +249,67 @@ def virtualenv(
...
-            "virtual environments. Use virtualenv for "
-            "your virtual environment to run integration "
-            "tests"
-        )
-    execfile(str(activate_this_f), dict(__file__=str(activate_this_f)))
+    else:
+        yield ''
 
 
 def pytest_addoption(parser):
```
Comment (strickvl) [inline tests/conftest.py:308]:
```suggestion
def pytest_add_option(parser):
```

Should this be like this?

File: tests/conftest.py [hunk 2]
```diff
@@ -314,6 +320,12 @@ def pytest_addoption(parser):
         default=False,
         help="Only run Kubeflow",
     )
+    parser.addoption(
```
Comment (strickvl) [inline tests/conftest.py:323]:
```suggestion
    parser.add_option(
```

Same thing here?

**Review by strickvl — APPROVED** [review]
Thank you!

File: tests/conftest.py (hunk 1 above)
Comment (AlexejPenner) [reply tests/conftest.py:308]: This function is recommended like this in the pytest [docs](https://docs.pytest.org/en/6.2.x/example/simple.html) - for ease of use and for finding resources I would leave it like this .

**Pull request merged**